presto-docs/src/main/sphinx/functions/khyperloglog.rst

```diff
@@ -0,0 +1,75 @@
+======================
+KHyperLogLog Functions
+======================
+
+Presto implements the `KHyperLogLog <https://research.google/pubs/pub47664/>`_
+algorithm and data structure. KHyperLogLog data structure can be created
+through :func:`khyperloglog_agg` function.
+
+
+Data Structures
+---------------
+
+KHyperLogLog is a data sketch that compactly represents the association of two
+columns. It is implemented in Presto as a two-level data structure composed of
+a MinHash structure whose entries map to HyperLogLog sketches.
+
+Serialization
+-------------
+
+KHyperLogLog sketches can be cast to and from ``varbinary``. This allows them to
+be stored for later use.
+
+Functions
+---------
+
+.. function:: khyperloglog_agg(x, y) -> KHyperLogLog
+
+    Returns the KHyperLogLog sketch that represents the relationship between
+    columns ``x`` and ``y``. The MinHash structure summarizes ``x`` and the HyperLogLog
+    sketches represent ``y`` values linked to ``x`` values.
+
+.. function:: cardinality(khll) -> bigint
+    :noindex:
+
+    This calculates the cardinality of the MinHash sketch, i.e. ``x``'s cardinality
+
+.. function:: intersection_cardinality(khll1, khll2) ->  bigint
+
+    Returns the set intersection cardinality of the data represented by the MinHash
+    structures of ``khll1`` and ``khll2``.
+
+.. function:: jaccard_index(khll1, khll2) ->  double
+
+    Returns the Jaccard index of the data represented by the MinHash structures of
+    ``khll1`` and ``khll2``.
+
+.. function:: uniqueness_distribution(khll) ->  map<bigint,double>
+
+    For a certain value ``x'``, uniqueness is understood as how many ``y'`` values are
+    associated to it in the source dataset. This is obtained with the cardinality
+    of the HyperLogLog that is mapped from the MinHash bucket that corresponds to
+    ``x'``. This function returns a histogram that represents the uniqueness
+    distribution, the X-axis being ``uniqueness`` and the Y-axis being relative
+    frequency of ``x`` values.
+
+.. function:: uniqueness_distribution(khll, histogramSize) ->  map<bigint,double>
+
+    Returns the uniqueness histogram with the given amount of buckets. If omitted,
+    the value defaults to 256. All ``uniqueness``es greater than ``histogramSize`` are
```

Comment:
looks like typo

Reply:
Not sure it's a typo, but does `All uniqueness values greater than` sound better?

Reply:
Yes, I think that would be more readable.